src/rtl/listbox.prg

```diff
@@ -408,13 +408,13 @@ METHOD hitTest( nMRow, nMCol ) CLASS ListBox
       RETURN nHit
    ENDIF
 
-   IF ! ::lIsOpen .OR. Empty( ::cHotBox + ::cColdBox )
-      nRet := 0
-   ELSE
-      IF ::lDropDown
-         nTop++
-      ENDIF
+   IF ::lIsOpen .AND. ::lDropDown
+      nTop++
+   ENDIF
+
+   IF ::lIsOpen .AND. .NOT. Empty( ::cHotBox + ::cColdBox )
```

Comment:
If you could rewrite this line to be ```IF ::lIsOpen .AND. ! Empty( ::cHotBox + ::cColdBox )```
and add some comment/note that here we're extending over Cl*pper's abilites, then I think your patch is okay to be pulled in.